nautobot/docs/user-guide/administration/configuration/index.md

```diff
@@ -149,11 +149,11 @@ echo $NAUTOBOT_CONFIG
 ### Adding your own dependencies
 
 !!! warning
-    Be cautious not to confuse extra applications with Nautobot plugins which are installed using the [`PLUGINS`](optional-settings.md#plugins) setting. They are similar, but distinctly different!
+    Be cautious not to confuse extra applications with Nautobot Apps which are installed using the [`PLUGINS`](optional-settings.md#plugins) setting. They are similar, but distinctly different!
 
 Nautobot, being a Django application, allows for installation of additional dependencies utilizing the [`INSTALLED_APPS`](https://docs.djangoproject.com/en/stable/ref/settings/#std:setting-INSTALLED_APPS) settings. Due to the highly specialized nature of Nautobot, *you cannot safely do this*.
 
-For example, let's assume that you want to install the popular [`django-health-check`](https://django-health-check.readthedocs.io/en/latest/) plugin to your Nautobot deployment which requires you to add one or more `health_check` entries to your `INSTALLED_APPS`.
+For example, let's assume that you want to install the popular [`django-health-check`](https://django-health-check.readthedocs.io/en/latest/) application to your Nautobot deployment which requires you to add one or more `health_check` entries to your `INSTALLED_APPS`.
```

Comment:
I'd probably stick with plugin here but it's not something I'm willing to argue any further than this one comment 😆

Reply:
Django calls them `apps` not `plugins`, generally speaking. :-)